app/controllers/concerns/sufia/collections_controller_behavior.rb

```diff
@@ -77,5 +87,9 @@ def form_class
       def _prefixes
         @_prefixes ||= super + ['catalog']
       end
+
+      def update_members_indices
+        params[:batch_document_ids].each { |gf| Sufia.queue.push(ResolrizeGenericFileJob.new(gf)) } unless params[:batch_document_ids].nil? || Sufia.config.collection_facet.nil?
```

Comment:
This is low priority, but would you consider reformatting this a bit using a guard clause and a coercion to an array?

```ruby
return if Sufia.config.collection_facet.nil?
Array(params[:batch_document_ids]).each do |gf|
  Sufia.queue.push(ResolrizeGenericFileJob.new(gf))
end
```